webapp/src/Migrations/Version20200214195715.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Do we still need this with the new configuration system? If a setting does not exist in the `etc/db-config.yaml` then it automatically does not show up anymore in the webinterface, right? So we don't need to delete anything, and this also has the (marginal) advantage that if you upgrade and then downgrade again, that your old customized settings are preserved.

Reply:
Yes, come to think of it, the migration is now wholly unnecessary

Reply:
Yep correct